### PR TITLE
docs: give prominence to Apptainer instead of Singularity

### PR DESCRIPTION
**Pull request by CanmingCobble**
## Description

Update docs to give prominence to Apptainer instead of Singularity

## Checklist

- [ ] Changes have been manually QA'd
- [ ] User-facing API changes need the "User-facing API Change" label.
- [ ] Release notes should be added as a separate file under `docs/release-notes/`.
  See [Release Note](https://github.com/determined-ai/determined/blob/master/docs/release-notes/README.md) for details.
- [ ] Licenses should be included for new code which was copied and/or modified from any external code.

### CODE REVIEW

File: docs/reference/deploy/config/master-config-reference.rst [hunk 1]
```diff
@@ -500,10 +500,10 @@ created by the launcher. The default is ``ALL``. The specification ``!root`` is
 appended to this list to prevent privilege elevation. See the ``sudoers(5)`` definition of
 ``Runas_List`` for the full syntax of this value. See :ref:`sudo_configuration` for details.
 
-``singularity_image_root``
---------------------------
+``apptainer_image_root`` or ``singularity_image_root``
```
Comment (jerryharrow) [inline docs/reference/deploy/config/master-config-reference.rst:503]: To add this you need to bump the launcher version we reference /setup-cluster/deploy-cluster/slurm/overview

File: docs/reference/deploy/config/master-config-reference.rst [hunk 2]
```diff
@@ -500,11 +500,11 @@ created by the launcher. The default is ``ALL``. The specification ``!root`` is
...
+------------------------------------------------------
 
-The shared directory where Singularity images should be located. This directory must be visible to
-the launcher and from the compute nodes. See :ref:`slurm-image-config` for more details.
+The shared directory where Apptainer/Singularity images should be located. This directory must be
+visible to the launcher and from the compute nodes. See :ref:`slurm-image-config` for more details.
```
Comment (jerryharrow) [inline docs/reference/deploy/config/master-config-reference.rst:507]: Need to say that only one of these two attributes can be specfied.

**Review by jerryharrow — COMMENTED** [review]
Need to update the section title in
setup-cluster/deploy-cluster/slurm/singularity.html#configuring-a-singularity-image-cache-directory

**Review by jerryharrow — COMMENTED** [review]
Need to update the section title in
setup-cluster/deploy-cluster/slurm/singularity.html#configuring-a-singularity-image-cache-directory

File: docs/reference/deploy/config/master-config-reference.rst (hunk 1 above)
Comment (CanmingCobble) [reply docs/reference/deploy/config/master-config-reference.rst:503]: Fixed.

File: docs/setup-cluster/deploy-cluster/slurm/slurm-requirements.rst [hunk 3]
```diff
@@ -275,13 +275,13 @@ interacts with PBS, we recommend the following steps:
...
 
 ************************************
- Singularity/Apptainer Requirements
+ Apptainer/Singularity Requirements
 ************************************
 
-Singularity/Apptainer is the recommended container runtime for Determined on HPC clusters. Apptainer
+Apptainer/Singularity is the recommended container runtime for Determined on HPC clusters. Apptainer
 is a fork of Singularity 3.8 and provides both the ``apptainer`` and ``singularity`` commands. For
 purposes of this documentation, you can consider all references to Singularity to also apply to
```
Comment (jerryharrow) [inline docs/setup-cluster/deploy-cluster/slurm/slurm-requirements.rst:283]: > purposes of this documentation, you can consider all references to Singularity to also apply to Apptainer

I would have thought we would have reversed this statement.  

Previously we always just said "Singularity" and it was intended to mean both.   Did we change all instance of "Singularity" to "Apptainer" such that we should reverse the sentence?

Comment (CanmingCobble) [reply docs/setup-cluster/deploy-cluster/slurm/slurm-requirements.rst:283]: But not all Apptainer commands apply to Singularity.

**Review by jerryharrow — APPROVED** [review]

**Comment by jerryharrow** [comment]
I'll review after you submit and see if we need any more follow-ups.

**Comment by CanmingCobble** [comment]
> I'll review after you submit and see if we need any more follow-ups.

Thanks.

**Pull request merged**